client/src/components/loading/BoardDetailsInfoSkeleton.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { Skeleton } from "@/components/ui/skeleton";
+import { BoardHeader } from "@/components/user-boards/BoardHeader";
+
+const SkeletonBoardDetails: React.FC = () => {
+  return (
+    <section className="rounded-lg border border-border bg-card p-4 shadow">
+      {/* Header */}
+      <header className="flex items-center justify-between gap-6 p-2">
+        <BoardHeader
+          title="Board Details"
+          description="Detailed information regarding everything connected to this board"
+        />
+        <Skeleton className="h-8 w-32 rounded-md" />
+      </header>
+
+      {/* Main Content */}
+      <main className="mt-4 flex flex-col flex-wrap gap-24 p-2 lg:flex-row">
+        {/* Left Column */}
+        <article className="flex flex-col lg:flex-[2]">
+          {/* Board Name Skeleton */}
+          <div className="mb-4 flex flex-col gap-1">
+            <div className="flex items-center">
+              <Skeleton className="mr-1 h-4 w-4" />
```

Comment:
I think we can get rid of these little skeletons. These were supposed to represent these little icons on the left side of labels, right?